### ISSUE TITLE
[nix-odml-conversion] Data Type Errors

### ISSUE THREAD

**Issue by fschrader1992** [achilleas-k/odml-to-nix#26]
When converting nix-files, errors like the following occur occasionally:

`TypeError: No NumPy equivalent for TypeBitfieldID exists`

This is a problem with the nix-core library, currently these properties are left out of the conversion. 
Should we achieve conversion of them as well and how?

**Comment by achilleas-k** [comment]
Do you have an example for a file that causes this?  My guess is these are very old files, back when the boolean representation in the file was a bitfield that h5py couldn't read.  Here's the relevant issue: https://github.com/G-Node/nixpy/issues/212

I think this change happened in NIXPy 1.2.1 or 1.3.0, so I expect the files will be from 2016 or older.

**Comment by fschrader1992** [comment]
Yes, indeed the files showing this error have been created before 2016. An example file is contained in pull request #27. Should we simply not support conversion of these files or is there a way to render them readable for h5py?

**Comment by achilleas-k** [comment]
I think h5py is still unable to read those, so for now I'd just mark them as unsupported.  Maybe at some point we can write a version upgrade tool that uses NIX (the C++ version) to convert them.
